src/test/chain-simulator/pool.cs-e2e.ts

```diff
@@ -30,20 +28,13 @@ describe('Pool e2e tests with chain simulator', () => {
         `${config.apiServiceUrl}/pool`,
       );
       const pool = response.data;
-      let numTimesAliceWasSender = 0;
-      let numTimesBobWasReceiver = 0;
       for (const tx of pool) {
-        numTimesAliceWasSender += tx.sender === config.aliceAddress ? 1 : 0;
-        numTimesBobWasReceiver += tx.receiver === config.bobAddress ? 1 : 0;
         expect(tx).toHaveProperty('receiver');
```

Comment:
instead of using .toHaveProperty you can add all the props into array and then use for loop to check if the response contain the props

Reply:
done